final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,52 @@
+import feed
+import feedparser
+import ssl
+import logging
+import exceptions as ex
+import check_func as check
+from args_creater import arguments
+
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
A question:

For what reason do you need this?

Reply:
without it feedparser.parse() return empty feed on MacOS